src/Middleware/Session/src/DistributedSession.cs

```diff
@@ -96,6 +96,10 @@ public string Id
         get
         {
             Load();
+            if(IsAvailable == false)
+            {
+                throw new InvalidOperationException("Session not currently available");
+            }
```

Comment:
The IsAvailable getter already calls `Load()` internally.
If the plan is to leave this check in place, let's remove the `Load()` from the line above.